lib/embed/gpi_embed.c

```diff
@@ -50,6 +50,22 @@ static char *argv[] = { progname };
 
 static PyObject *pEventFn = NULL;
 
+/**
+ * @name    Print the traceback
+ * @brief   Print traceback to stderr
+ *
+ * GILState before calling: N/A
+ *
+ * GILState after calling: N/A
+ *
+ * Print traceback and flush stderr if required (on Windows)
+ */
+void print_traceback(void) {
+  PyErr_Print();
+#if defined(_WIN32)
+  PyRun_SimpleString("import sys; sys.stderr.flush()");
```

Comment:
@stuarthodgson Yes, for some reasons under Modelsim on Windows, it's required to call the flush from the Python interpreter.

@LinuxFanatic777 The fix corrects the issue when there is an problem in the Python code prior to Python code execution. I was not aware this issue was also there on Linux. I have no way to test it on Modelsim SE under Linux.